content/en/style-guide/_index.md

```diff
@@ -176,6 +178,90 @@ More about that in the [How To Contribute](/contribute/) doc.
 Before getting started, please read some published Glossary terms 
 to get a feeling for the level of detail and difficulty and when examples are appropriate.
 
+
+
+## Sign Language Video Style Guide {#sign-language-style-guide}
+
+If you are contributing a sign language video, please follow these guidelines to ensure our videos are clear, informative, and accessible to a wide audience (Including hearing community members).
+
+
+### Technical equipment
+
+All you need is a high-resolution camera and a stand. A modern smartphone should be enough.
+
+
+### Recording Setup
+
+
+
+* **Lighting:** Frontal light
+
+
+* **Background:** Record in front of a flat, solid background so it doesn't distract from the signing, and ensure no distracting decorations are included.
+
+
+* **Dress Code:** Solid single-color top (no prints) contrasting your skin tone.
+
+
+* **Camera Position:** Head to elbow should be on the screen and the camera should be positioned near the eye's height. The camera should also be perpendicular to the background (the background behind you should not be slanted).
+
+
+* **Signing Space:** Use as much space as possible, ensuring you have full signing space while sitting or standing.
+
+
+* **Sound:** Mute the microphone (input audio) when possible to avoid involuntary background noise.
+
+### Signing
+
+
+
+* **Clarity and Accuracy:** Record the sign twice: frontally and from the side, and sign slowly enough, so non-signers can see and mimic it. 
+
+
+* **Facial Expression:** Use relaxed or neutral facial expressions. Mouthing or making grimaces is strongly discouraged. Maintain your eyes on the camera if possible.
+
+
+* **Frequency:** Only one sign is enough for each video as playback can be looped. Repeating the sign in a video is unnecessary.
+
+
+* **Duration:** The total length of the video should not be more than 7 seconds.
+
+
+* **Fingerspelling:** Use sign language for the word as much as possible as fingerspelling is the last resort, commonly reserved for abbreviated or short words.
+
+
+* **Posture:** While signing, only maintain frontal orientation with eyes facing the camera while positioned. Demonstrating signs using side orientation is unnecessary. When at rest, arms should be down with hands relaxed.
```

Comment:
### Signing

- **Clarity and Accuracy:** Record the sign twice from a frontal orientation, signing slowly enough for non-signers to see and mimic it. Demonstrating signs from the side is unnecessary unless it adds clarity.

- **Facial Expression:** Use natural facial expressions and avoid exaggerations or grimaces that might distract. Maintain eye contact with the camera if possible.

- **Frequency:** Only one sign is needed for each video, as playback can be looped. Repeating the sign within a video is unnecessary.

- **Duration:** The total length of the video should not exceed 7 seconds.

- **Fingerspelling:** Use sign language for the word as much as possible. Fingerspelling is a last resort, commonly reserved for abbreviated or short words.

- **Posture:** Maintain a frontal orientation with eyes facing the camera while signing. When at rest, arms should be down with hands relaxed.